modules/apps/portal-language/portal-language-override-web/src/main/java/com/liferay/portal/language/override/web/internal/portlet/action/ImportTranslationsMVCActionCommand.java

```diff
@@ -83,32 +77,12 @@ private void _importTranslations(
 			return;
 		}
 
-		Properties languageProperties = new Properties();
-
-		languageProperties.load(
-			new InputStreamReader(
-				new FileInputStream(file), StandardCharsets.UTF_8));
-
-		if (languageProperties.size() == 0) {
-			SessionErrors.add(actionRequest, "fileInvalid");
```

Comment:
It looks like in moving this, we have lot the "fileInvalid" sessionError. Is there a reason for this, or should it be added back?

Reply:
Good catch @ethib137. In this case, the local service should throw, and the action command should catch and populate the SessionErrors.

Reply:
Oops, my bad. Just fixed it, I added a new exception `InvalidPropertiesFileException`